src/components/Header/index.js

```diff
@@ -2,17 +2,15 @@ import React from 'react'
 import { Link } from 'react-router-dom'
 
 import routes from '../../routes'
+import { getItem } from '../../localStorage'
 
 import './styles.css'
 
 const Header = () => {
   return (
     <section className="header">
       <div className="headerInner">
-        <Link
-          to={localStorage.getItem('userLogin') ? routes.profile : routes.signUp}
-          className="headerProfile"
-        >
+        <Link to={getItem('userLogin') ? routes.profile : routes.signUp} className="headerProfile">
```

Comment:
as i said before, it would be better if you could clear JSX from complex expressions like
```
getItem('userLogin') ? routes.profile : routes.signUp
```

more convenient way for this is:
```
...
const route = getItem('userLogin') ? routes.profile : routes.signUp
...
<Link to={route} ... />
...
```